File: C++/Program/Produk.cpp
#include <iostream>
#include <string>

using namespace std;

class Produk
{
	private:
		string nama;
		string id;
		string brand;
		string price;
	
	public:
	// construct
	Produk()
	{
		this->nama = "";
		this->id = "";
		this->brand = "";
		this->price = "";
	}
	
	Produk(string nama, string id, string brand, string price)
	{
		this->nama = nama;
		this->id = id;
		this->brand = brand;
		this->price = price;
	}
	/* setter */
	void setNama(string nama)
	{
		this->nama = nama;
	}
	
	void setId(string id)
	{
		this->id = id;
	}
	
	void setBrand(string brand)
	{
		this->brand = brand;
	}
	
	void setPrice(string price)
	{
		this->price = price;
	}
	/* getter */
	string getNama()
	{
		return this->nama;
	}
	
	string getId()
	{
		return this->id;
	}
	
	string getBrand()
	{
		return this->brand;
	}
	
	string getPrice()
	{
		return this->price;
	}	
};

File: Python/Program/Produk.py
class Produk:
    __nama = ""
    __id = ""
    __brand = ""
    __price = ""
    
    def __init__ (self, name = "", id = "", brand = "", price = ""):
        self.__nama = name
        self.__id = id
        self.__brand = brand
        self.__price = price
        
    # getter #
    def getNama(self):
        return self.__nama
        
    def getId(self):
        return self.__id
    
    def getBrand(self):
        return self.__brand
    
    def getPrice(self):
        return self.__price
    
    # setter #
    def setNama(self, name):
        self.__nama = name
        
    def setId(self, id):
        self.__id = id
    
    def setBrand(self, brand):
        self.__brand = brand
    
    def setPrice(self, price):
        self.__price = price
    